src/etos_lib/eiffel/common.py

```diff
@@ -27,7 +27,7 @@
 
 PUBLISHER_TEMPLATE = "{SERVER_ADDRESS}:{SERVER_PORT},{RABBITMQ_VHOST},{RABBITMQ_EXCHANGE}"
 CONSUMER_TEMPLATE = "{RABBITMQ_QUEUE_NAME}"
-# pylint:disable=too-many-arguments
+# pylint:disable=too-many-arguments,too-many-positional-arguments
```

Comment:
Is the old `too-many-arguments` necessary now with the new `too-many-positional-arguments` ?

Reply:
Yes, both are necessary.